ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py

```diff
@@ -0,0 +1,50 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.tensorflow as ivy_tf
+
+
+@given(
+    y_true=st.lists(st.integers(min_value=0, max_value=5), min_size=1, max_size=1),
+    y_pred=helpers.dtype_and_values(
+        available_dtypes=ivy_tf.valid_float_dtypes,
+        shape=(5,),
+        min_value=-10,
+        max_value=10,
+    ),
+    from_logits=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.sparse_categorical_crossentropy"
+    )
+)
+def test_sparse_categorical_crossentropy(
+        y_true,
+        y_pred,
+        from_logits,
+        num_positional_args,
+        fw
+):
+    y_true = np.array(y_true, dtype='int32')
+    dtype = y_pred[0]
```

Comment:
you can do this instead:
dtype, y_pred = dtype_y_pred
and use dtype_y_pred to pass the result of helpers.dtype_and_values
in this way, y_pred is naturally a ivy.array and dtype can be used later for the input_dtypes argument.
Please check the docstring of the relative functions and look for examples in other test files.